e2e/rbd.go

```diff
@@ -1240,6 +1240,14 @@ var _ = Describe("RBD", func() {
 				validateRBDImageCount(f, 0)
 			})
 
+			By("create a PVC and Bind it to an app for mapped rbd image with options", func() {
+				deleteResource(rbdExamplePath + "storageclass.yaml")
+				createRBDStorageClass(f.ClientSet, f, nil, map[string]string{"mapOptions": "lock_on_read,queue_depth=1024"})
```

Comment:
I'd add an unmap option here to cover `unmapOptions` stashing code as well.

Reply:
as you said earlier it's not good idea to use `force` so I have not added unmap options. let me update the storageclass for force options

Reply:
It's fine to use in a E2E test.  It's a supported option after all -- just one that is rarely useful (and particularly in the CSI case, where the details of mapping and unmapping are hidden from the end user).

Reply:
added now.